src/go/cmd/testdata/test_struct/test.go

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package teststruct
```

Comment:
Add internal struct that doesn't show up?

Reply:
Done